src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,48 @@
+package lotto.domain;
+
+public enum Rank {
+    NONE(0, 0, false),
+    FIFTH(3, 5_000, false),
+    FOURTH(4, 50_000, false),
+    THIRD(5, 1_500_000, false),
+    SECOND(5, 30_000_000, true),
+    FIRST(6, 2_000_000_000, false);
+
+    private final int matchCount;
+    private final int prizeMoney;
+    private final boolean bonusMatch;
+
+    Rank(int matchCount, int prizeMoney, boolean bonusMatch) {
+        this.matchCount = matchCount;
+        this.prizeMoney = prizeMoney;
+        this.bonusMatch = bonusMatch;
+    }
```

Comment:
(이렇게 바꿔보는 건 어때요?)
```suggestion
    NONE(0, 0),
    FIFTH(3, 5_000),
    FOURTH(4, 50_000),
    THIRD(5, 1_500_000),
    SECOND(5, 30_000_000,true),
    FIRST(6, 2_000_000_000);

    private final int matchCount;
    private final int prizeMoney;
    private final boolean bonusMatch;

    Rank(int matchCount, int prizeMoney) {
        this.matchCount = matchCount;
        this.prizeMoney = prizeMoney;
        this.bonusMatch = false;
    }
    Rank(int matchCount, int prizeMoney, boolean bonusMatch) {
        this.matchCount = matchCount;
        this.prizeMoney = prizeMoney;
        this.bonusMatch = bonusMatch;
    }
```
true조건은 하나만 존재하니깐 생성자를 2개 만드는 것도 좋은 방법이라고 생각해요 :)